app/controllers/rubric_criteria_controller.rb

```diff
@@ -2,7 +2,7 @@ class RubricCriteriaController < ApplicationController
 
   before_action :authorize_only_for_admin
 
-  def download_csv
+  def csv_download
```

Comment:
Don't rename this in this pull request. (You can rename it later when you're actually working on this method.)